packages/fluentui/react-northstar/src/components/Accordion/AccordionTitle.tsx

```diff
@@ -137,37 +136,34 @@ class AccordionTitle extends UIComponent<WithAsProp<AccordionTitleProps>, any> {
   renderComponent({ ElementType, classes, unhandledProps, styles, accessibility }) {
     const { contentRef, children, content, indicator, contentWrapper } = this.props;
 
-    const contentWrapperElement = (
-      <Ref innerRef={contentRef}>
```

Comment:
This is suspicious, I had to remove the Ref and add the ref `contentRef` directly on the Box in order to solve the issue..